tests/util_tests/test_shotgun_connect.py

```diff
@@ -365,9 +386,21 @@ def _run_test(self, site, source_proxy, source_store_proxy, expected_store_proxy
             self.assertEqual(sg.base_url, self._SITE)
             self.assertEqual(sg.config.raw_http_proxy, source_proxy)
 
+            from sgtk.util.shotgun import connection
+            shotgun_yml_data = connection.internal_get_associated_sg_config_data(
+                "unused_core_location_because_auth_user_is_set"
+            )
+
+            # FIXME: We're losing the difference between the value being absent
+            # and the value forced to None 
```

Comment:
Here are the failing tests I was mentioning earlier. Note that a few other tests are missing.